### DISCUSSION TITLE
Documentation of global settings

### DISCUSSION THREAD

**Discussion by kieran-osgood** [TanStack/query#1236] [Ideas]
**Describe the Issue**
Pretty new to react-query so I've been reading through documentation as I go and I've combed through trying to adjust the default settings for all query retries, not sure if I'm just skipping the part its described, but as an example I saw a reference to setting global settings for [query-retries](https://react-query.tanstack.com/docs/guides/query-retries), but then couldn't find anywhere that references how to do this specifically until I looked through the API Reference and eventually saw the `new QueryCache({config})`. 

I'm not sure this will be immediately obvious to new comers to the library (...like me).

**Suggestion**
I'm thinking of more documentation on the [Important Defaults](https://react-query.tanstack.com/docs/guides/important-defaults) to add a reference to either a page describing it or just to the api reference where you override the defaults in `new QueryCache({...config})`, and add links from places like the /query-retries page that talk about global settings

Happy to PR the adjustments, though if I'm just being blind (happens all too often) and missing the part it is described then please point me in the correct direction!

**Comment by GenhaoLi** [comment]
Now it seems `new QueryCache({config})` does **not** provide any config for global default options. The correct way is probably using `new QueryClient({defaultOptions})`. 

However, I do agree the doc should include a reference to this, either in page [important defaults](https://tanstack.com/query/latest/docs/framework/react/guides/important-defaults) or [query options](https://tanstack.com/query/latest/docs/framework/react/guides/query-options)
